### DISCUSSION TITLE
Clicking the Publish Button Does Nothing

### DISCUSSION THREAD

**Discussion by captinwill** [WilliamVenner/gmpublisher#165] [Help]
I can't make this more simple to understand. When I click the "Publish!" button it does nothing. The app says nothing, no error pops up, nothing.

**Comment by WilliamVenner** [comment]
is the button greyed out or blue?

**Comment by captinwill** [comment]
its grey

**Comment by WilliamVenner** [comment]
Then you need to fill out more details about the addon. Tags, category, etc
